internal/services/wallets/wallets_pubnet.go

```diff
@@ -15,6 +15,18 @@ var PubnetWallets = []data.Wallet{
 			},
 		},
 	},
+	{
+		Name:              "Vibrant Assist RC",
+		Homepage:          "vibrantapp.com/vibrant-assist",
```

Comment:
Not very happy about this workaround. Currently homepages are unique in the wallets table, but in prd we'll need to support both vibrant and vibrant rc.

Reply:
😅